registry/tests/utils.py

```diff
@@ -85,17 +84,17 @@ def cb(request):
             if auth is None:
                 return (401, {}, "Not logged in")
             else:
+                # Hack to make it easy to simulate admin users:
+                # if the username contains "admin", it's an "admin".
+                is_staff = 'admin' in auth
```

Comment:
This seems a bit dangerous, or at least confusing. E.g., 'notadmin' would actually be treated as an admin user by that rule. Could we just have a field that contains the admin username? Or, a list of names? Even a match of the exact string 'admin' would seem safer ('admin' == vs 'admin' in auth).